src/OpenTelemetry.Exporter.Jaeger/JaegerExporterOptions.cs

```diff
@@ -14,14 +14,41 @@
 // limitations under the License.
 // </copyright>
 
+using System;
 using System.Diagnostics;
+using OpenTelemetry.Exporter.Jaeger.Implementation;
 
 namespace OpenTelemetry.Exporter
 {
     public class JaegerExporterOptions
     {
         internal const int DefaultMaxPayloadSizeInBytes = 4096;
 
+        private const string OTelAgentHostEnvVarKey = "OTEL_EXPORTER_JAEGER_AGENT_HOST";
+        private const string OTelAgentPortEnvVarKey = "OTEL_EXPORTER_JAEGER_AGENT_PORT";
+
+        public JaegerExporterOptions()
+        {
+            string agentHostEnvVar = Environment.GetEnvironmentVariable(OTelAgentHostEnvVarKey);
+            if (!string.IsNullOrEmpty(agentHostEnvVar))
+            {
+                this.AgentHost = agentHostEnvVar;
+            }
+
+            string agentPortEnvVar = Environment.GetEnvironmentVariable(OTelAgentPortEnvVarKey);
+            if (!string.IsNullOrEmpty(agentPortEnvVar))
+            {
+                if (int.TryParse(agentPortEnvVar, out var agentPortValue))
+                {
+                    this.AgentPort = agentPortValue;
+                }
+                else
+                {
+                    JaegerExporterEventSource.Log.FailedToParseEnvironmentVariable(OTelAgentPortEnvVarKey, agentPortEnvVar);
```

Comment:
@cijothomas Should we actually throw here? I think typically we do for misconfiguration during startup.

Reply:
My answer would be no.

It is currently unspecified according to the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/7fc28733eb3791ebcc98fed0d858a7961f1e95b2/specification/sdk-environment-variables.md#numeric-value).

Looking at the other part of the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/7fc28733eb3791ebcc98fed0d858a7961f1e95b2/specification/sdk-environment-variables.md#enum-value), it seems the general spirit is that the SDK should handle environment variable in a graceful way (e.g. if some environment variable wouldn't make sense, fall back to the default value rather than fail the app).

I think the reason is that environment variable might be invasive (e.g. a child process might inherit environment variable, someone might set global environment variable unintentionally), and we don't want that to crash the app.

Reply:
Just wondering in the case of a typo how visible will be that? Are the host and port logged in case of error sending to a destination? Perhaps log the host, port (all the options in general) at startup so one can easily check the effective configuration?

Reply:
@cijothomas What do you think of logging the options as an event with `EventLevel.Informational` (or even `EventLevel.IVerbose`) e.g. here: https://github.com/open-telemetry/opentelemetry-dotnet/blob/a5b42fbe171895025ed57d685951d9c734940a04/src/OpenTelemetry.Exporter.Jaeger/JaegerExporter.cs#L52 ?

Reply:
As long as we don't log credential by default, I'm supportive (Information/Verbose both sound good to me) 😃
* If the environment variable is something like `OTEL_EXPORTER_JAEGER_AGENT_PASSWORD`, we should not log it by default (or we should replace the password with something like `******`).
* If the user put credential in `OTEL_EXPORTER_JAEGER_AGENT_PORT`, it's their problem, we will log it (rather than trying to do some smart detection and _guess_ if it could be credential).

Reply:
@reyang @cijothomas 
Should I do it as a separate PR?
Personally, I see it as a separate feature and maybe it would be worth adding a dedicated point in the changelog for it.

Reply:
+1 to doing it as a separate PR.